Libplanet/Net/TxCompletion.cs

```diff
@@ -0,0 +1,191 @@
+#nullable enable
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Tx;
+using Nito.AsyncEx;
+using Serilog;
+
+namespace Libplanet.Net
+{
+    public class TxCompletion<TPeer, TAction> : IDisposable
```

Comment:
Why do we need `TPeer` type parameter for `TxCompletion`? At look glance, there is only `BoundPeer` for that.

Reply:
This refers to `BlockCompletion`... I'm not really don't know why it used like this though.

Reply:
🤔